<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankAutomat</class>
 <widget class="QMainWindow" name="BankAutomat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BankAutomat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color rgb(20, 27, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	font: 20pt &quot;Helvetica Neue&quot;;
	color: #fff;
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: rgb(0, 150, 255);
    min-width: 80px;
}

QLabel
{
	font: 18pt &quot;Helvetica Neue&quot;;
	color: #fff;
}
QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(211, 236, 255);
    selection-background-color: darkgray;
}
QWidget {     
background-color:rgb(0, 0, 255);
}</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QLineEdit" name="loginlineEditKorttiID">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>210</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>KorttiID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Helvetica Neue&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;h1&gt;Kirjaudu sisään&lt;/h1&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginBtnKirjaudu">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>340</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>Kirjaudu</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginlineEditTunnusluku">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>270</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1234</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tunnusluku</string>
      </property>
     </widget>
     <widget class="QLabel" name="loginLabelInfo">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>420</y>
        <width>561</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 175, 0);</string>
      </property>
      <property name="text">
       <string>&lt;h1&gt; OttoMaatti&lt;/h1&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DepitCreditPage">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>170</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;h2&gt;Valitse&lt;/h2&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DebitCreditBtnDebit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>280</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>Debit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DebitCreditBtnCredit">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>Credit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 175, 0);</string>
      </property>
      <property name="text">
       <string>&lt;h1&gt;OttoMaatti&lt;/h1&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Actionpage">
     <widget class="QPushButton" name="ActionBtnOtto">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>170</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>Otto</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ActionBtnSaldo">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>240</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>Saldo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ActionBtnTapahtumat">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>310</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>Tapahtumat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;h2&gt;Valitse&lt;/h2&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>311</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 175, 0);</string>
      </property>
      <property name="text">
       <string>&lt;h1&gt;OttoMaatti&lt;/h1&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="saldopage">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;h2&gt;Tilin tilanne&lt;/h2&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>230</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Tilin saldo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saldoBtnLopeta">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>360</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>Lopeta</string>
      </property>
     </widget>
     <widget class="QLabel" name="saldoLabelTilinsaldo">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>230</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>60 euroa</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saldoBtnPalaa">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>360</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>Palaa</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>331</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 175, 0);</string>
      </property>
      <property name="text">
       <string>&lt;h1&gt;OttoMaatti&lt;/h1&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="withdrawpage">
     <widget class="QPushButton" name="withdrawBtn20">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
     <widget class="QPushButton" name="withdrawBtn40">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>40</string>
      </property>
     </widget>
     <widget class="QPushButton" name="withdrawBtn50">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>180</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>50</string>
      </property>
     </widget>
     <widget class="QPushButton" name="withdrawBtn100">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>270</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QPushButton" name="withdrawBtnPaluu">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>480</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>Paluu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;h2&gt;Otto&lt;/h2&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>291</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 175, 0);</string>
      </property>
      <property name="text">
       <string>&lt;h1&gt;OttoMaatti&lt;/h1&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="withdrawBtn20_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>470</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>Muu summa</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>341</width>
        <height>201</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>311</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;strong&gt;Näppäile summa&lt;/strong&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="withdrawBtnOk">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 137, 255);</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="withdrawlineEditAmount">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="withdrawLabelInfo">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>341</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Transactionpage">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>341</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;h2&gt;Tilin tapahtumat&lt;/h2&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="transactionLabelDate">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>181</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string>01.01.1990&lt;br&gt;01.01.1990&lt;br&gt;01.01.1990&lt;br&gt;01.01.1990&lt;br&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TransactionBtnLopeta">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>420</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>Lopeta</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TransactionBtnPalaa">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>420</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 137, 255);</string>
      </property>
      <property name="text">
       <string>Palaa</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>291</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 175, 0);</string>
      </property>
      <property name="text">
       <string>&lt;h1&gt;OttoMaatti&lt;/h1&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="transactionLabelSum">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>210</y>
        <width>181</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string>90 euroa&lt;br&gt;90 euroa&lt;br&gt;90 euroa&lt;br&gt;90 euroa&lt;br&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="transactionLabelTransact">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>200</y>
        <width>181</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string>Pano&lt;br&gt;Pano&lt;br&gt;Pano&lt;br&gt;Pano&lt;br&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Endingpage">
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>451</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;h2&gt;Ota rahat, ole hyvä.&lt;/h2&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>390</y>
        <width>731</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;h3&gt;Kiitos käynnistä, tervetuloa uudelleen!&lt;/h3&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>651</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/img/Pankkimaattikuva.png);
widht:300px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
